app/models/framework/definition/RM3754.rb

```diff
@@ -14,14 +21,14 @@ class Invoice < EntryData
         field 'Customer Organisation', :string, exports_to: 'CustomerName', presence: true
         field 'Customer Invoice Date', :string, exports_to: 'InvoiceDate', ingested_date: true
         field 'Customer Invoice Number', :string, exports_to: 'InvoiceNumber', presence: true
-        field 'Product Description', :string, exports_to: 'ProductDescription'
-        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }, allow_nil: true
-        field 'Unit of Purchase', :string, exports_to: 'UnitType'
-        field 'Price per Unit', :string, exports_to: 'UnitPrice', ingested_numericality: true, allow_nil: true
-        field 'Vehicle Registration No', :string, exports_to: 'Additional1'
-        field 'Total Number of Units', :string, exports_to: 'UnitQuantity', ingested_numericality: true, allow_nil: true
+        field 'Product Description', :string, exports_to: 'ProductDescription', presence: true
+        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }
+        field 'Unit of Purchase', :string, exports_to: 'UnitType', presence: true
+        field 'Price per Unit', :string, exports_to: 'UnitPrice', ingested_numericality: true
+        field 'Vehicle Registration No', :string, exports_to: 'Additional1', presence: true
+        field 'Total Number of Units', :string, exports_to: 'UnitQuantity', ingested_numericality: true
         field 'Total Charge (ex VAT)', :string, exports_to: 'InvoiceValue', ingested_numericality: true
-        field 'Payment Profile', :string, exports_to: 'Additional2'
+        field 'Payment Profile', :string, exports_to: 'Additional2', case_insensitive_inclusion: { in: PAYMENT_PROFILES }
```

Comment:
Just an observation - should we consider fixing the case of any entries like this during ingest? (e.g. "monthly" -> "Monthly"). I suspect it'll make the data warehouse easier to query too.